var/spack/repos/builtin/packages/hypre/package.py

```diff
@@ -257,6 +264,8 @@ def configure_args(self):
                 configure_args.append("--with-cuda-home={0}".format(spec["cuda"].prefix))
             else:
                 configure_args.append("--enable-cub")
+            if "+cublas" in spec:
+                conigure_args.append("--enable-cublas")
```

Comment:
```suggestion
            if spec.satisfies("+cublas"):
                conigure_args.append("--enable-cublas")
```

It's now recommended to use `spec.satisfies()` instead of `in`.